### PR TITLE
Translate stock_filter_none_zero_qty module to spanish

### PR DESCRIPTION
**Pull request by alan196**

### CODE REVIEW

File: stock_filter_none_zero_qty/i18n/es.po [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 1.8.7\n"
+"Last-Translator: Jesús Alan Ramos Rodríguez <alan.ramos@jarsa.com.mx>\n"
+"Language: es\n"
+
+#. module: stock_filter_none_zero_qty
+#: code:addons/stock_filter_none_zero_qty/product.py:55
+#, python-format
+msgid "Operator %s not suported in searches for qty_available (product.product)."
+msgstr "El operador %s no es soportado en búsquedas de qty_available (product.product)."
```
Comment (pedrobaeza) [inline stock_filter_none_zero_qty/i18n/es.po]: s/es/está

Comment (alan196) [inline stock_filter_none_zero_qty/i18n/es.po]: I don't get it. 
"El operador %s no está soportado en búsquedas de qty_available (product.product)."?
This is what you want?

Comment (pedrobaeza) [inline stock_filter_none_zero_qty/i18n/es.po]: Yeah, that's it

**Comment by luistriana28** [comment]
:+1:

**Comment by nhomar** [comment]
+1

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**